bucket/komorebi.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.1.7",
+    "description": "A tiling window manager for Windows",
+    "homepage": "https://github.com/LGUG2Z/komorebi",
+    "license": "MIT",
+    "notes": [
+        "Run 'cp $original_dir\\komorebi.sample.ahk $env:USERPROFILE\\komorebi.ahk' to get started with the sample configuration.",
+        "Run 'komorebic ahk-library' if you would like to generate an AHK helper library to use in your configuration.",
+        "Once you have a configuration file in place, you can run 'komorebic start' to start the window manager."
+    ],
+    "suggest": {
+        "autohotkey": "extras/autohotkey"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/LGUG2Z/komorebi/releases/download/v0.1.7/komorebi-0.1.7-x86_64-pc-windows-msvc.zip",
+            "hash": "dc7d22d7a3b3bae46c11f734c92e4c4c61a1eb5e3c344fd06cc4361b259067b5"
+        }
+    },
+    "pre_install": "if (Get-Process -Name komorebi -ErrorAction SilentlyContinue) { komorebic stop }",
+    "post_install": [
+        "Write-Host \"Copying 'komorebi' to shims folder.\"",
+        "Copy-Item $dir\\komorebi.exe $scoopdir\\shims\\komorebi.exe -Force"
+    ],
+    "uninstaller": {
+        "script": "Remove-Item $scoopdir\\shims\\komorebi.exe"
+    },
+    "bin": [
+        "komorebic.exe"
+    ],
```

Comment:
```suggestion
    "bin": "komorebic.exe",
```

Reply:
Actually, would it be good if I added `komorebi.exe` to `bin` so it can still be registered as a "shim"?

Reply:
Then you will need the force parameter while copying

Reply:
It works without the `-Force` parameter.